roiextractors/segmentationextractor.py

```diff
@@ -16,7 +16,10 @@ class SegmentationExtractor(ABC, BaseExtractor):
 
     def __init__(self):
         BaseExtractor.__init__(self)
-        self._sampling_frequency = None
+        self._sampling_frequency = np.float('NaN')
+        self._channel_names = ['OpticalChannel']
+        self._raw_movie_file_location = ''
+        self.no_planes = 1
```

Comment:
A `SegmentationExtractor` can only have 1 plane. Why specify it?

Reply:
its only for now as nwbwriter looks for this, but once the Multisegmentation is implemented, then this would go